draft-looker-oauth-jwt-cwt-status-list.md

```diff
@@ -366,7 +366,72 @@ TODO evaluate deifnition of Status List Provider?
 
 # IANA Considerations
 
-This document specifies no IANA actions.
+## JSON Web Token Claims Registration
+
+This specification requests registration of the following Claims in the
+IANA "JSON Web Token Claims" registry [@IANA.JWT] established by [@!RFC7519].
+
+*  Claim Name: `status`
```

Comment:
Given most jwt claims try to informally follow the three character convention, should we consider 'sts'? I dont feel strongly.

Reply:
https://github.com/vcstuff/draft-looker-oauth-jwt-cwt-status-list/issues/40